jerry-core/ecma/base/ecma-globals.h

```diff
@@ -204,7 +204,7 @@ typedef enum
 
   ECMA_INTERNAL_PROPERTY_NATIVE_HANDLE, /**< native handle associated with an object */
   ECMA_INTERNAL_PROPERTY_FREE_CALLBACK, /**< object's native free callback */
-
+  ECMA_INTERNAL_PROPERTY_NOT_FOUND_AND_STOP, /**< the property is not found and no more search */
```

Comment:
That is convincing. Thank you for the example. Then we definitely need a way to stop searching.